hooks/mkdocs/G003_social_cards_adapter.py

```diff
@@ -0,0 +1,152 @@
+#! python3  # noqa: E265
+
+"""Script joué au chargement du contenu markdown de chaque page (hook).
+
+Contexte :
+
+    Le thème Material for Mkdocs intègre un plugin (social) qui permet de générer
+    automatiquement une image pour chaque page en croisant différents éléments
+    (métadonnées de la page et éléments graphiques de base).
+    Mais le fonctionnement est très générique (normal) et entre en conflit avec la gestion
+    des en-têtes du site Geotribu.
+
+Objectifs :
+
+    - quand une image n'est pas définie sur une page, patcher l'en-tête pour
+        y insérer la configuration du plugin social de sorte qu'il génère une image
+        et qu'on la référence
+    - quand une image est définie manuellement, patcher l'en-tête pour
+        y insérer la configuration du plugin social de sorte qu'il utilise l'image sans rien toucher
+
+"""
```

Comment:
Excuse my french, but I don't understand this.

Reply:
No excuse.
Well I though that it's more important to keep the files identical to maintain smooth copy/paste in the future as I did to make this change. Or maybe I'm too lazy to translate it.

Reply:
Okay I see, it makes sense and I agree, Sir Moura.
Vive le baguette !